Source/Core/Core/HotkeyManager.cpp

```diff
@@ -36,7 +36,7 @@ const std::string hotkey_labels[] = {
     _trans("Start Recording"),
     _trans("Play Recording"),
     _trans("Export Recording"),
-    _trans("Read-only mode"),
+    _trans("Read-Only mode"),
```

Comment:
Mode should be capitalized, I think I accidentally missed it the first time around. I will fix that.